### ISSUE TITLE
[question] Curious on removed features

### ISSUE THREAD

**Issue by ffuentesm** [jsmnbom/ao3-enhancements#80]
I assume the removed "TrackWorks" was that feature that showed if a fic was bookmarked to subscribe in the search page? 
![image](https://github.com/user-attachments/assets/8e861410-f427-4eb4-889c-81278f518627)

Kind of sad, since this was a time-saving feature for when I searched for new fics to read via tag search. 
It easily let me know if I was already reading that fic or not. Since I bookmark every fic I read as a way to track. 

Is there no way to maybe add an extra stat in the fic column, like a "Subscribed" or "Bookmarked" instead of the icons from the previous version?
Like this:
![image](https://github.com/user-attachments/assets/ee65b23e-880d-4534-b26a-d7cb1741418b)

**Comment by jsmnbom** [comment]
Hi

This feature required loading each fic page in the backhground and checking if you had subscribed to it or given kudos and such. This ended up adding a bunch of random fics to user histories which is not great so I decided to get of it.

Unfortunately I've been able to find a stable and good solution to detect if you've given kudos to a work and such that does not require loading the actual fic page. AO3 currently lacks any API, and I can't find the data on any other page, so until such a time that AO3 adds an api, this feature probably will not be returning.
